UPDATING.md

```diff
@@ -445,9 +445,9 @@ The following table shows changes in import paths.
 |airflow.contrib.operators.gcp_compute_operator.GceInstanceStopOperator                                            |airflow.gcp.operators.compute.GceInstanceStopOperator                                                      |
 |airflow.contrib.operators.gcp_compute_operator.GceInstanceTemplateCopyOperator                                    |airflow.gcp.operators.compute.GceInstanceTemplateCopyOperator                                              |
 |airflow.contrib.operators.gcp_compute_operator.GceSetMachineTypeOperator                                          |airflow.gcp.operators.compute.GceSetMachineTypeOperator                                                    |
-|airflow.contrib.operators.gcp_container_operator.GKEClusterCreateOperator                                         |airflow.gcp.operators.kubernetes_engine.GKEClusterCreateOperator                                           |
-|airflow.contrib.operators.gcp_container_operator.GKEClusterDeleteOperator                                         |airflow.gcp.operators.kubernetes_engine.GKEClusterDeleteOperator                                           |
-|airflow.contrib.operators.gcp_container_operator.GKEPodOperator                                                   |airflow.gcp.operators.kubernetes_engine.GKEPodOperator                                                     |
+|airflow.contrib.operators.gcp_container_operator.ComputeEngineCreateClusterOperator                               |airflow.gcp.operators.kubernetes_engine.ComputeEngineCreateClusterOperator                                           |
```

Comment:
This looks like a misunderstanding. I have already updated the spreadsheet because these operators should still have the prefix "GKE"
https://docs.google.com/spreadsheets/d/1nRI31B6t3PR7Ljsw6BSPqHkoPGzN11BeYEQRXsjEAWU/edit#gid=1271422594

Reply:
Yes, my mistake. Commit is on his way.

Reply:
@michalslowikowski00 I think we want to have
```
airflow.gcp.operators.container.GKECreateClusterOperator
airflow.gcp.operators.container.GKEDeleteClusterOperator
```
instead of 
```
airflow.gcp.operators.container.ComputeEngineCreateClusterOperator
airflow.gcp.operators.container.ComputeEngineDeleteClusterOperator
```
@mik-laj

Reply:
Aw... you are right. Whole PR is wrong. 
Damn... Ok, I will close this one and make the new one.